docs/content/file-formats.md

```diff
@@ -61,6 +62,39 @@ spec:
 
 [CredentialSet]: /operator/glossary/#credentialset
 
+## ParameterSet
+
+See the glossary for more information about the [ParameterSet] resource.
+
+```yaml
+apiVersion: porter.sh/v1
+kind: ParameterSet
+metadata:
+  name: parameterset-sample
+spec:
+  schemaVersion: 1.0.1
+  namespace: operator
+  name: porter-test-me
+  parameters:
+    - name: test-secret
+      source:
+        value: test-value
+    - name: test-secret
+      source:
+        secret: test-secret
+```
+
+| Field                     | Required | Default                            | Description                                                 |
+|---------------------------|----------|------------------------------------|-------------------------------------------------------------|
+| agentConfig               | false    | See [Agent Config](#agentconfig)   | Reference to an AgentConfig resource in the same namespace. |
+| porterConfig              | false    | See [Porter Config](#porterconfig) | Reference to a PorterConfig resource in the same namespace. |
+| parameters                | true     |                                    | List of parameter sources for the set |
+| parameters.name           | true     |                                    | The name of the parameter for the bundle |
+| parameters.source         | true     |                                    | The paremeters type. Currently `vaule` and `secret` are the only supported sources |
+| parameters.source.(value or secret)   | true     |                                    | The plaintext value to use or the name of the secret that holds the parameter |
```

Comment:
Should `oneOf` be used.  Not sure if json schema aligns with OpenApi on this.

Reply:
Yeah I wasn't sure what to put here so threw this in. I'll look into OpenAPI spec on how to do this, this was just placeholder